docs/components/zeebe/technical-concepts/internal-processing.md

```diff
@@ -57,17 +57,23 @@ This command can in turn be processed, completing the service task and driving t
 
 ## Handling backpressure
 
+import BackpressureWarning from '../../../components/react-components/backpressure-warning.md'
+import BackpressureExport from '../../../components/react-components/backpressure-export.md'
+
 When a broker receives a client request, it is written to the **event stream** first, and processed later by the stream processor. If the processing is slow or if there are many client requests in the stream, it might take too long for the processor to start processing the command. If the broker keeps accepting new requests from the client, the backlog increases and the processing latency can grow beyond an acceptable time.
 
-To avoid such problems, Zeebe employs a [backpressure](/self-managed/zeebe-deployment/operations/backpressure.md) mechanism.
-When the broker receives more requests than it can process with an acceptable latency, it rejects some requests.
+<BackpressureExport/>
+
+To avoid such problems, Zeebe employs a [backpressure](/self-managed/zeebe-deployment/operations/backpressure.md) mechanism. When the broker receives more requests than it can process with an acceptable latency, it rejects some requests.
```

Comment:
Non-blocking suggestion: "uses" instead of "employs" (plain English/better for translation down the line)